database/seed-data/fakeDataGenerator.js

```diff
@@ -1,37 +1,58 @@
 const faker = require('faker');
 
+// genereate fake question
 const fakeQuestion = () => faker.lorem.sentence().slice(0, -1) + '?';
-// generate an array of 1-10 answers
-const fakeAnswers = () => {
+
+// generate fake user
+const fakeUser = () => faker.name.findName();
+
+// generate fake date
+const fakeDate = () => faker.date.past();
+
+// generate rating between 1-100
+const fakeRating = () => Math.floor(Math.random() * 100) + 1;
+
+// generate an array of 1-10 answers, each with an answer, user, and date keys
+const fakeAnswers = async () => {
   let numOfAnswers = Math.floor(Math.random() * 10) + 1;
   let answersArray = [];
+
   while (numOfAnswers > 0) {
-    answersArray.push(faker.lorem.paragraph());
+    answersArray.push({
+      answer: await faker.lorem.paragraph(),
+      user: await fakeUser(),
+      date: await fakeDate()
+    });
+
     numOfAnswers--;
   }
 
   return answersArray;
 };
-const fakeUser = () => faker.name.findName();
-const fakeDate = () => faker.date.past();
-// generate rating between 1-100
-const fakeRating = () => Math.floor(Math.random() * 100) + 1;
 
 const fakeDataGenerator = async (recordsToCreate, startingRecordID) => {
   let data = [];
 
+  // generate N number of records based on param
   while (recordsToCreate > 0) {
+    // generate random number between 1-10 for number of questions per product
```

Comment:
I like your logic in generating a different amount of questions for each product.